### ISSUE TITLE
Is it possible to detach/unload/unhook from script

### ISSUE THREAD

**Issue by 0xABD** [frida/frida#68]
1) Is it possible to remove all the installed hooks and completely detach Frida from a process by calling a command from script? 

The use case is when auditing partially self-checking/self-modifying binaries to make the necessary instrumentations before the self-checking/self-modifying code finds them, save the results somewhere and then detach, but leave the process running with minimum modifications.

2) Is the same possible from frida-trace? There is an Interceptor.detachAll() method, but it looks like it does not work at all if called from within frida-trace `__handlers__` scripts.

**Comment by oleavr** [comment]
Sorry about the late follow-up, spent some time offline.

1) Not currently; you are meant to send() a message to the host-side and have it call `.detach()` on the session (or `.destroy()` on the script).

2) No. Patch would be very welcome though! :)

**Comment by zhendeshiming** [comment]
this.destroy() can destroy all ?

**Comment by oleavr** [comment]
Please open a PR against frida-tools if you're interested in this functionality.
